css/style.css

```
@@ -0,0 +1,230 @@
+/* 
+Spacing system (px)
+2 / 4 / 8 / 12 / 16 / 24 / 32 / 48 / 64 / 80 / 96 / 128
+
+Font sizes (px)
+10 / 12 / 14 / 16 / 18 / 20 / 24 / 30 / 36 / 44 / 52 / 62 / 74 / 86 / 98
+*/
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  overflow-x: hidden;
+}
+
+body {
+  line-height: 1;
+  overflow-x: hidden;
+}
+
+.promo {
+  max-width: 160rem;
+  margin: 0 auto;
+}
+
+/******************************/
+/* HEADER */
+/******************************/
+
+.header {
+  position: absolute;
+  display: flex;
+  width: 160rem;
+  height: 12.2rem;
+  justify-content: center;
+  align-items: center;
+  padding: 0 16.8rem;
+  background: linear-gradient(to bottom, black -100%, transparent 90%);
+  gap: 0.8rem;
+}
+
+.logo-menu-bar {
+  height: inherit;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: inherit;
+}
+
+.logo {
+  font-family: "Yeseva One", sans-serif;
+  text-decoration: none;
+  color: #fff;
+  text-transform: uppercase;
+  font-size: 3.6rem;
+  opacity: 0.9;
+  padding: 1rem;
+}
+
+/******************************/
+/* NAVIGATION */
+/******************************/
+
+.main-nav-list {
+  list-style: none;
+  display: flex;
+  gap: 3rem;
+}
+
+.main-nav-link,
+.main-nav-link:link,
+.main-nav-link:visited,
+.main-nav-link:active {
+  padding: 4.8rem 1rem;
+  position: relative;
+  display: inline-block;
+  text-decoration: none;
+  color: #fff;
+  font-style: normal;
+  font-size: 2.2rem;
+  font-weight: 600;
+  transition: all 0.3;
+  text-shadow: 1px 1px rgba(0, 0, 0, 0.3);
+  border-top: 0.38rem solid transparent;
+}
+
+.main-nav-link:hover,
+.main-nav-link:active {
+  opacity: 75%;
+  transition: 0.3s;
+}
+
+.main-nav-link.is-active {
+  opacity: 100%;
+  border-top: 0.38rem solid #fff;
+}
+
+.main-nav-link:focus {
+  opacity: 84%;
+  border-top: 0.38rem solid #fff;
+}
+
+.social-links {
+  list-style: none;
+  display: flex;
+  gap: 1.6rem;
+  padding-right: 3.6rem;
+  align-items: center;
+}
+
+.social-link:link,
+.social-link:visited {
+  display: flex;
+  text-decoration: none;
+  color: #dab33e;
+  transition: all 0.3s;
+  font-size: 2rem;
+  align-items: center;
+}
+
+.social-link:hover,
+.social-link:active {
+  opacity: 50%;
+  transition: 0.3s;
+}
+
+/******************************/
+/* PROMO SECTION */
+/******************************/
+
+.promo-section {
+  max-width: 160rem;
+  height: 75rem;
+}
+
+.promo-banner {
+  background: url('../gallery/banners/promo-banner.jpg');
+  background-size: cover;
+  max-width: 160rem;
+  height: 75rem;
+  padding-top: 12rem;
+  padding-bottom: 2rem;
+}
+
+.promo-slogan {
+  width: inherit;
+  height: 25rem;
+  margin-top: 10rem;
+}
+
+.slogan {
+  width: inherit;
+  padding: 0 2rem;
+  font-family: "Yeseva One", sans-serif;
+  text-transform: lowercase;
+  text-align: center;
+  font-size: 9rem;
+  line-height: 11.4rem;
+  font-weight: 100;
+  color: #F5F5F5;
```

Let's use lowercased variant for all colors
```suggestion
  color: #f5f5f5;
```

---

ok